ibm_catalog.json

```diff
@@ -21,20 +21,21 @@
         "ai"
       ],
       "short_description": "Creates and configures an instance of IBM watsonx.data.",
-      "long_description": "This architecture supports creating and configuring an instance of watsonx.data.",
+      "long_description": "This architecture supports creating and configuring an instance of watsonx.data. For more details on watsonx data, [see here](https://cloud.ibm.com/docs/watsonxdata/index.html).",
       "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-data/blob/main/README.md",
       "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-watsonx-data/main/images/watsonx-data.svg",
       "provider_name": "IBM",
       "features": [
         {
           "title": "Creates an instance of IBM watsonx.data",
-          "description": "Creates and configures an IBM watsonx.data instance."
+          "description": "For more details on Watsonx Data instances, [see here](https://cloud.ibm.com/watsonxdata#about)"
         },
         {
           "title": "Creates a KMS encryption enabled IBM watsonx.data instance",
           "description": "Create and configures a KMS encryption enabled IBM watsonx.data instance for Enterprise plan."
         }
```

Comment:
* Optional integration is missing. Already mentioned [here](https://github.ibm.com/GoldenEye/issues/issues/13592#issuecomment-112085880)
* The first point can be deleted as per previous discussion. Initiated the thread in cpre and lets see whats the final outcome.
* `see here` can be written as `refer here` / `refer this` / `learn more`